e2e/Makefile

```diff
@@ -41,6 +41,10 @@ functional-test: install-godog
 	STACK_VERSION=${STACK_VERSION} \
 	godog --format=${FORMAT} ${FEATURE_FLAG} ${FEATURE}
 
+.PHONY: lint
+lint:
+	@docker run -ti --rm -v $(PWD):/src -w /src gherkin/lint **/*.feature --disable AvoidOutlineForSingleExample,TooClumsy,TooManySteps,TooManyDifferentTags,TooLongStep
```

Comment:
Will it be possible to run the pre-commit in the CI? If so, we don't need this particular make goal

Reply:
Yes, it's still possible to run pre-commit in the CI, I wanted to have an easy manner to run it locally, setting the exclusions and getting a better output:

<img width="1005" alt="Screenshot 2020-08-18 at 18 24 33" src="https://user-images.githubusercontent.com/951580/90539432-186bc180-e180-11ea-9a7b-4e32875776a2.png">